### PR TITLE
Evaluate method added to DDPG, PPO1, VPG

### PR DESCRIPTION
**Pull request by Zeus3101**
Fix for #38 . Can be easily extended by copy-pasting the same code to other algos.

### CODE REVIEW

File: jigglypuffRL/deeprl/ddpg/ddpg.py [hunk 1]
```diff
@@ -209,6 +209,30 @@ def learn(self):
...
+            a = self.select_action(s)
+            s1, r, done, _ = env.step(a)
+            ep_r += r
+            ep_t += 1
+
+            if done:
+                ep += 1
+                print("Ep: {}, reward: {}, t: {}".format(ep, ep_r, ep_t))
+                if self.tensorboard_log:
+                    self.writer.add_scalar("eval_episode_reward", ep_r, ep_t)
```
Comment (Sharad24) [inline jigglypuffRL/deeprl/ddpg/ddpg.py:227]: I'm not sure if we need TB logs in the evaluate method. @ajaysub110 Do we need this?

I feel we can avoid this.

Comment (ajaysub110) [reply jigglypuffRL/deeprl/ddpg/ddpg.py:227]: Yeah agree. Evaluate is just to observe output right? We can have just reward printing and environment rendering

**Comment by Zeus3101** [comment]
I guess we can make it a common method in utils.py under deeprl. Would that be fine? Because the code needn't be different for each of them

File: jigglypuffRL/deeprl/ddpg/ddpg.py (hunk 1 above)
Comment (Zeus3101) [reply jigglypuffRL/deeprl/ddpg/ddpg.py:227]: Okay, I wasn't sure if it was required so I put it anyway.

**Comment by Sharad24** [comment]
Yup, should work.

**Comment by Sharad24** [comment]
Just the common function comment, then I think we can have this PR merged as well. @Zeus3101 Please also make sure about to have a class variable signature of the function I.e., `self.evaluate = evaluate`

**Comment by Zeus3101** [comment]
Please check the new commit. Made evaluate a common function in utils.py under jigglypuffRL.common

**Comment by Sharad24** [comment]
Looks like there are some conflicts, can you resolve these?

**Comment by Zeus3101** [comment]
Resolved changes and removed evaluate from parameters descriptions

**Pull request merged**